section2/reflection.md

```diff
@@ -2,14 +2,58 @@
 
+- To be honest, it was a bit off putting. it was like having a tricycle for 2 minutes and then being thrown on a motorcycle (exaggerated but still lol)
+However, it really pushed me to try to comprehend the instructions more and do research of my own. When i was able to figure it out, it was just a big "AHA!" moment.
+I do notice that the examples that are being provided in the exercises are being reduced and it made me worried on whether or not i was doing the assignment right.
+But again, it pushed my to actually think instead of looking at the examples and trying to make my answer work/match.
+This was especially prominent if the "if-statements" exercise when we were asked to make a condition for pizza. Looking back, i had a lot of fun just
+trying to figure how to structure things and how to make it work.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+-A conditional statement is "if something happens then this should happen. if not, then this happens".
+One example of this is: On amazon, if i buy a anime figure then I will get it shipped to my house.
+If i don't get it, then nothing will happen and i save money.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+- You can add multiple conditions to an `if` statement by doing `else if` and then stating a new condition. You can keep doing that but it will probably look messy and your computer might take a bit to figure out what's going on because, if the first condition is not true then it moves on to the next one and sees if it's true or not. You computer will keep checking each condition until it finds a true statement and when it does, it stops and executes that line of code and whatever else you want it to do in that situation.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+- You can use logical operators: The AND operator `&&` is used when you need two or more conditions to be met.
+The OR operator `||` is used when you need only one of the two or more conditions to be met.
+
+- In addition, to the logical operators, you can use the `dev console` in a browser (in the console tab) to run code and call variables to see what the output will be before
+running a block of code.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+- `if` statements are fine but they seem a bit...messy. if i had 20 conditions, that would take forever to code. is there a faster way?
```

Comment:
There are case statements, that is one way to assign a lot of conditions that is a little cleaner than an if statement. There are also enums, where you can use a number to represent a value that can be assigned. If statements are usually best used with just a few conditions.